bodhi/server/models.py

```diff
@@ -158,12 +266,10 @@ def grid_columns(cls):
 
     def update_relationship(self, name, model, data, db):
         """Add items to or remove items from a many-to-many relationship
-
```

Comment:
Dropping all of these spaces are not related to the change - can you make sure that only the ```git mv``` and merging enum in are happening in this pull request?